api/xds_protocol.rst

```diff
@@ -613,6 +613,27 @@ adding/removing/updating clusters. On the other hand, routes are not
 warmed, i.e., the management plane must ensure that clusters referenced
 by a route are in place, before pushing the updates for a route.
 
+.. _xds_protocol_TTL:
+
+TTL
+~~~
+
+In the event that the management server becomes unreachable, the last known configuration received by envoy will persist until the connection is reestablished. For some services, this may not be desirable. For example, in the case of a fault injection service, a management server crash at the wrong time may leave envoy in an undesirable state. The TTL setting allows envoy to remove a set of resources after a specified period of time if contact with the management server is lost. This can be used, for example, to terminate a fault injection test when the management server can no longer be reached.
```

Comment:
Another good example is stale endpoint assignments, where endpoints which are intended to receive less load, e.g. unhealthy, continue to get hammered on management server disconnect.

Another good use case is caching (either via xDS relay or on local disk).